.github/workflows/test-schema.yaml

```
@@ -1,6 +1,6 @@
 name: Test OWASP Schema
 
-on:
+true:
```

actionlint (1.7.4)

unexpected key "true" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)